docs/README.md

```diff
@@ -1,38 +1,41 @@
 # Open Active Conformance Services
 
-Various tools for harvesting and normalising OpenActive Opportunity feeds to a common representation
+A suite of tools to discover, download, normalise, enhance and republish data that uses the OpenActive specification. 
 
-See the [docs](/docs/) folder for more information.
+See the [docs](./) folder for more information.
 
```

Comment:
This link to docs won't work in both github and gitbook but happy to prioritise gitbook as in github you can easily see the docs dir.